react/w-books/src/app/routers/AppRouter.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+import Login from '~screens/Login';
+import SignUp from '~screens/SignUp';
+
+export const AppRouter = () => (
```

Comment:
By conventions of the Department export your component like:

` function AppRouter () {...} export default AppRouter`